charts/logentries/templates/daemonset.yaml

```diff
@@ -16,6 +16,7 @@ spec:
     metadata:
       labels:
         app: {{ template "logentries.name" . }}
+        daemonset: {{ template "logentries.name" . }}
```

Comment:
it's the same values as used for label `app`. does it make sense to have this label then?

Reply:
actually, it would be enough if we just use app, if they both have the same values :) [Changed it](d6aa107).
I adjusted the [NOTES.txt](881eef7) accordingly. Thanks for the catch :)